index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7967e81f77.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+   <div class="container">
+       <header>
+           <nav class="navItems">
+            <h1>Welcome</h1>
+                    <i class="fa-solid fa-bars"></i>
+           </nav>
+       </header>
+    <section class="headline">
+           <h1>Hey there.I'm <br> Precious <br>
+               <span> I'm a software developer</span>
+           </h1>
+           <h3 id="headline_text"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            do not hestiate to contact me.
+           </h3>
```

Comment:
- [ ] Great job, you have done a big work but let's make a few changes; as for here, please use a `p` tag instead of this h3,